.github/workflows/test.yaml

```diff
@@ -168,14 +168,14 @@ jobs:
           # extension.
           # There is very little PYPY specific code so there is not much to
           # gain from reporting coverage.
-          - python-version: 'pypy-3.9'
+          - python-version: 'pypy3.10-v7.3.13'
             tox-env: 'alldeps-nocov-posix'
             job-name: 'no-coverage'
             skip-coverage: yes
 
           # We still run some tests with coverage,
           # as there are test with specific code branches for pypy.
-          - python-version: 'pypy-3.9'
+          - python-version: 'pypy3.10-v7.3.13'
```

Comment:
Maybe mention in the releas notes that PyPy3.9 is no longer supported, and we now support PyPy 3.10?

Reply:
The interesting thing that is happening here is that we are not supporting the latest PyPy, 7.3.14; 7.3.13 works with both 3.9 and 3.10, and the CPython coverage for syntax differences seems sufficient, given that those two "versions" are substantially similar. So I wouldn't say it's unsupported, and if we have PyPy-3.9 specific issues we could add another buildbot. I bumped the Python version since if we are going to make space in the test matrix, it seems like 3.10 would probably be more used at this point than 3.9.